cheroot/server.py

```diff
@@ -2056,14 +2068,22 @@ def interrupt(self):
         """Flag interrupt of the server."""
         return self._interrupt
 
+    @property
+    def _stopping_for_interrupt(self):
+        """Return whether the server is responding to an interrupt."""
+        return self._interrupt is _STOPPING_FOR_INTERRUPT
+
     @interrupt.setter
     def interrupt(self, interrupt):
-        """Perform the shutdown of this server and save the exception."""
-        self._interrupt = True
+        """Perform the shutdown of this server and save the exception.
+
+        Typically invoked by a worker thread in threadpool.py,
```

Comment:
Here's how you can reference the module in Sphinx
```suggestion
        Typically invoked by a worker thread in
        :py:mod:`~cheroot.workers.threadpool`,
```
(there's pages autogenerated from docstrings @ https://cheroot.cherrypy.org/en/latest/pkg/cheroot.server.html#cheroot.server.HTTPServer.interrupt)